lib/CModHandler.h

```diff
@@ -14,6 +14,12 @@
 #include "VCMI_Lib.h"
 #include "JsonNode.h"
 
+#ifdef __UCLIBC__
+#undef major
+#undef minor
+#undef patch
+#endif

```

Comment:
Could you please elaborate why it's needed? And where __UCLIBC__ is defined?

Reply:
uClibc is a small standard C library like glibc or musl. And this stdlib have these macro defined :(
This is why I have to undefine them.

Reply:
wouldn't it be better to rename variables in VCMI code to avoid the clash?

Reply:
I just took the path of least change.
Let me know if you want to use different variable names instead.

Reply:
@kambala-decapitator 
Nah, I think it's fine.
Should we also ban usage of "min" and "max" because windows.h has those defines?

Personally I'd call this bug of uglibc since library exposes its internal defines using names that are not classified as reserved.

Reply:
@IvanSavenko fair :)